centos/6/Dockerfile

```diff
@@ -1,25 +1,38 @@
 FROM centos:6
 MAINTAINER Roman Tsisyk <roman@tarantool.org>
 
+# The only available devtoolset-7 in Vault archives found, check:
+#   https://vault.centos.org/centos/6/sclo/x86_64/rh/Packages/d/
+ARG DEVTOOLSET=devtoolset-7
+
+# Removed missing CentOS-Base repositories
+ADD CentOS-Base.repo /etc/yum.repos.d/
```

Comment:
Looks a bit strange: the comment is about removing, but the command is `ADD`. I would say something like 'replace dead mirror.centos.org with vault.centos.org'.

Reply:
Fixed.